database/repositories/interests.php

```diff
@@ -21,4 +21,26 @@ function get_all_interests(): array
     return array_column($result->fetch_all(), 0);
 }
 
+function get_user_interests(string $user): array|null // base64
```

Comment:
Can you change the parameter to $user_id. We need to agree on when we use the word user, we refer to the ID. That way we dont need to specify in the function name 
Also can you remove the // at the end